src/App.js

```diff
@@ -1,9 +1,54 @@
 import './App.css';
-
+import Grid from '@mui/material/Grid';
+import Notes from './components/Notes';
+import { Container } from '@mui/material';
+import Heading from './components/Heading';
+import BasicModal from './components/Modal';
+import BottomNavigation from '@mui/material/BottomNavigation';
+import Paper from '@mui/material/Paper'
+import notesList from './data.json';
+import { LocationCity } from '@mui/icons-material';
 function App() {
   return (
-    <div className="App">
+    <div>
+    <Heading />
+    <Container sx={{marginTop: 5, marginBottom: 12, }}>
+    {notesList.map((notes) => (
+
+<Grid container spacing={5}> 
+   
+{notes.exampleNotes.map((note,index) => <Notes note={note} key={index} />)}
```

Comment:
here as well. Also, why do we have a notes array and inside that another array?
In my opinion, we should have only one array of notes